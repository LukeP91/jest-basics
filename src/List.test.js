import React from 'react';
import List from './List.js';
import renderer from 'react-test-renderer';

describe('List', () => {
  test('renders correctly', () => {
    const list = ["todo 1", "todo 2"]
    const tree = renderer
      .create(<List list={list}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
