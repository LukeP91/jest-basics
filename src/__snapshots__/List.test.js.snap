// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <div
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "whitesmoke",
        "marginBottom": 5,
        "padding": 15,
      }
    }
  >
    todo 1
  </div>
  <div
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "whitesmoke",
        "marginBottom": 5,
        "padding": 15,
      }
    }
  >
    todo 2
  </div>
</div>
`;
